test_assertive_pyupgrade.py

```
@@ -0,0 +1,151 @@
+# pyupgrade (UP) - catches upgrade opportunities
+def test_pyupgrade():
+    # UP001: Use X instead of Y (various Python version upgrades)
+    from typing import List, Dict, Tuple  # UP001
```

Ruff (0.12.2)

`typing.List` imported but unused

Remove unused import

(F401)